lib/ui/widgets/post_content/widgets/link_preview/index.dart

```diff
@@ -0,0 +1,97 @@
+import 'package:flutter/material.dart';
+import 'package:mooncake/ui/ui.dart';
+import 'package:url_launcher/url_launcher.dart';
+import './utils/preview_helper.dart';
+
+class LinkPreview extends StatefulWidget {
```

Comment:
Can you please add the documentation to this widget in order to make sure that new developers understand when/where to use it properly and what data to pass to it as well?